src/controllers/historycontroller.js

```diff
@@ -0,0 +1,47 @@
+class HistoryController extends Controller
+{
+    static addEvent(event)
```

Comment:
No, has nothing to do with jQuery events.
I thought it was kind of logical to use `HistoryController.addEvent(event)` in any other controller, and pass any `Customer/Container/FactoryOrder` with it, to store it in the `history` array.